src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2509,62 +2509,64 @@ ITermDispatch::StringHandler AdaptDispatch::RequestSetting()
 // - None
 void AdaptDispatch::_ReportSGRSetting() const
 {
+    using namespace std::string_view_literals;
+
     // A valid response always starts with DCS 1 $ r.
     // Then the '0' parameter is to reset the SGR attributes to the defaults.
-    std::wstring response = L"\033P1$r0";
+    fmt::basic_memory_buffer<wchar_t, 64> response;
```

Comment:
is 64 long enough?

Reply:
Yeah I was counting with each number being 2 digits and all parameters present and I got 56 chars.
For comparison: `std::wstring` comes equipped with a whopping 7 chars for the SSO buffer.